_includes/tutorials/anomaly-detection/ksql/markup/dev/create-suspicious-names-table.adoc

```diff
@@ -0,0 +1,5 @@
+First, you will need to create a ksqlDB table and Kafka topic to represent the suspicious names data. A table is more fitting for this suspicious names data because it is a mutable collection that changes over time. We may want to add company names to this table or remove them in the future.
+
```

Comment:
Other workflows sometimes presume pre-existence of the Kafka topic before creating the ksqlDB stream or table.  Would it help to add a note about what happens in this case where the Kafka topic doesn't already exist?

Reply:
I added a little blurb about kafka topics that should address your comment (below)
>First, you will need to create a ksqlDB table and Kafka topic to represent the suspicious names data. You can create a table from a Kafka topic or derive one from an existing stream or table. In both cases, a table's underlying data is durably stored in a topic on the Kafka brokers. In this tutorial we are creating a new Kafka topic for our table. If `kafka_topic` was not specified in the query, a new Kafka topic would be created for us.